cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ModelUtils.java

```diff
@@ -739,4 +739,15 @@ public static boolean hasOwnerReference(HasMetadata resource, OwnerReference own
     public static int caCertGeneration(Ca ca) {
         return Annotations.intAnnotation(ca.caCertSecret(), Ca.ANNO_STRIMZI_IO_CA_CERT_GENERATION, Ca.INIT_GENERATION);
     }
+
+    /**
+     * Returns the id of a pod given the pod name
+     *
```

Comment:
Is this space required?

Reply:
I think we normally have an empty lines between the description block, params block and return.